packages/venia-concept/summarize-validate-queries.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * This file outputs a summary blurb to the console informing the developer
+ * about PWA to Magento version compatibility.
+ *
+ * This will help developers experiencing compatibility issues to solve them.
+ */
+
+const semver = require('semver');
+
+const compatibilityDefinitions = require('../../magento-compatibility.js');
+const packageJSON = require('../../package.json');
+
+const DOCS_COMPAT_TABLE_PATH = 'https://magento-research.github.io/pwa-studio/';
```

Comment:
I propose we actually create a "PWA / Magento Compatibility" page on the docs site that pulls in the contents of `pwa-devdocs/src/_includes/magento-compatibility.md` (the compatibility table in markdown).

If we have that, we can update this URL to point directly to it.

Reply:
I support this 👍

Reply:
```suggestion
const DOCS_COMPAT_TABLE_PATH = 'https://magento-research.github.io/pwa-studio/technologies/magento-compatibility/';
```